src/edu/ccsu/cs505/hw2/CheckProcessor.java

```diff
@@ -0,0 +1,22 @@
+package edu.ccsu.cs505.hw2;
+
+/**
+ * @author Julie Shevchenko
+ * CheckProcessor class
+ **/
+public class CheckProcessor {
+
+    /**
+     * processes check
+     * @param acc first account in CoR
+     * @param amount the total amount to be deducted from customer's accounts
+     * catches InsufficientFundsException
+     **/
+    public void processCheck(Account acc, double amount) {
+        try {
+            acc.deduct(amount);
+        } catch (InsufficientFundsException e) {
+            e.printStackTrace();
```

Comment:
processCheck is supposed to return true/false so when you handle the exception it should return false and output message rather than a stack trace.